prometheus/build_info_collector.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build go1.12
```

Comment:
Should we have a dummy implementation for older versions, at least until this library no longer supports 1.11?

Reply:
Yes, much better. Otherwise, everybody has to do the gymnastics I did in the example.

So I'll add a dummy implementation and simplify the example.